src/ServiceControl.Monitoring.Data.Tests/ServiceControl.Monitoring.Data.approved.txt

```diff
@@ -1,42 +1,2 @@
 ﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("ServiceControl.Monitoring.Data.Tests")]
-[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
```

Comment:
Can you bump up the major @bording ? It's a totally different package after all :)

Reply:
Done!